cherryontech/jupiter-jumpers
# Storybook

## Description

This PR:
1) Integrates Tailwind with Storybook
2) Publishes storybook (https://662b31961b954f0f663b71e1-zxyuxxdhlp.chromatic.com/).
3) Sets up auto deployment of the published site when future changes are pushed (added chromatic.yml)
4) Updates the README to include a link to our published Storybook 

## Related Issue

N/A

## Related Story Card

N/A

## Type of Changes

Dependency update

## Acceptance Criteria

N/A

## Testing Instructions

1) Run `npm run storybook`, and check the button component: it should be blue (via the Tailwind CSS class)
2) Check https://662b31961b954f0f663b71e1-zxyuxxdhlp.chromatic.com/. There should be a published version of the Storybook.
